app/src/main/AndroidManifest.xml

```diff
@@ -84,6 +84,10 @@
       android:theme="@style/Clinic.V2.Theme.BottomSheetActivity"
       android:windowSoftInputMode="adjustResize" />
 
+    <activity
+      android:name=".setup.SetupActivity"
+      android:enabled="false"/>
```

Comment:
Curious, what does `android:enabled="false"` do?

Reply:
The activity cannot be started unless it is manually enabled. Basically meant to stop someone from accidentally starting it.